<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSettings</class>
 <widget class="QWidget" name="WidgetSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>425</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBoxLogin">
       <property name="title">
        <string>Credentials</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>User login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditLogin">
          <property name="maxLength">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPassword">
          <property name="maxLength">
           <number>255</number>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Let empty to take last stored password.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxRememberPw">
          <property name="toolTip">
           <string>Remember your login and sign in automatically.</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Sign in on application start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelLine">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>1</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>1</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
  background-color: rgb(131,147, 167);
}</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCreateAccount">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You are new? &lt;a href=&quot;REGISTER&quot;&gt;&lt;span style=&quot; text-decoration: underline; color: white;&quot;&gt;Regisgter an account!&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelForgotPassword">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;FORGOT_PW&quot;&gt;&lt;span style=&quot; text-decoration: underline; color: white;&quot;&gt;Forgot your password?&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>20</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButtonSignOut">
            <property name="toolTip">
             <string>Sign out from your account.</string>
            </property>
            <property name="text">
             <string>Sign out</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSignIn">
            <property name="toolTip">
             <string>Sign in into your account.</string>
            </property>
            <property name="text">
             <string>Sign in</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxAdvanced">
       <property name="title">
        <string>Advanced</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupBoxServer">
          <property name="styleSheet">
           <string notr="true">QGroupBox {
  border: 0;
}</string>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Application Server Address</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEditServer">
             <property name="maxLength">
              <number>255</number>
             </property>
             <property name="placeholderText">
              <string>[http:// or https://]&lt;Server Address&gt;:&lt;Port&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>onBtnSignInClicked()</slot>
  <slot>onBtnSignOutClicked()</slot>
 </slots>
</ui>
